next-app/package.json

```diff
@@ -2,6 +2,7 @@
     "name": "app",
     "version": "0.1.0",
     "private": true,
+    "type": "module",
```

Comment:
putting `module` here allows you to use es module syntax for your `next.config.js` - a minor upgrade imo